cmd/credentialUtil.go

```diff
@@ -139,17 +139,28 @@ func getBlobCredentialType(ctx context.Context, blobResourceURL string, canBePub
 // 1. If there is cached session OAuth token, indicating using token credential.
 // 2. If there is OAuth token info passed from env var, indicating using token credential. (Note: this is only for testing)
 // 3. Otherwise use shared key.
```

Comment:
Minor: need to either bring this comment into line with the revised code, or remove the comment. Given that BlobFS is more complex for auth options than anything else, retaining and updating the comment seems to make some sense. (Even tho normally we don't want to just restate the code in comment form).

Reply:
Code has the correct order of precendence (SAS before OAuth) but comment has it around the wrong wany.

---
In reply to: [287934924](https://github.com/Azure/azure-storage-azcopy/pull/401#discussion_r287934924) [](ancestors = 287934924)